apps/remote_control/lib/lexical/remote_control/project_node.ex

```diff
@@ -69,12 +73,25 @@ defmodule Lexical.RemoteControl.ProjectNode do
       %{state | status: :stopped}
     end
 
-    def on_nodeup(%__MODULE__{} = state) do
-      %{state | status: :started}
+    def on_nodeup(%__MODULE__{} = state, node_name) do
+      if node_name == Project.node_name(state.project) do
```

Comment:
Is this if-else mainly for `debug_shell`?

Reply:
Yes